antarest/study/storage/rawstudy/model/filesystem/config/links.py

```diff
@@ -159,11 +228,12 @@ def _validate_colors(cls, values: t.MutableMapping[str, t.Any]) -> t.Mapping[str
         return validate_colors(values)
 
     # noinspection SpellCheckingInspection
-    def to_config(self) -> t.Dict[str, t.Any]:
+    def to_ini(self, version: int) -> t.Dict[str, t.Any]:
```

Comment:
I see no usage of this method so it should be removed.

Reply:
I removed it and everything still works so I guess you're right